src/test/java/uk/gov/justice/hmpps/casenotes/controllers/ReadCaseNoteTypeIntTest.kt

```diff
@@ -20,61 +13,44 @@ class ReadCaseNoteTypeIntTest : ResourceTest() {
   }
 
   @Test
-  fun `default request provides inactive, active, non secure and non restricted types`() {
+  fun `default request provides inactive, active, non secure and restricted types`() {
```

Comment:
Dropping "non secure" i.e. `default request provides inactive, active and restricted types` possibly better explains the test purpose?